packages/pwa-kit-react-sdk/src/ssr/universal/errors.ts

```diff
@@ -6,11 +6,10 @@
  */
 
 export class HTTPError extends Error {
-    constructor(status, message) {
+    status: number
+
+    constructor(status: number, message: string) {
         super(message)
-        this.constructor = HTTPError
-        this.__proto__ = HTTPError.prototype
-        this.message = message
```

Comment:
Whats happening here specifically?

Reply:
Calling super already implicitly calls the constructor